framework/src/aifw/Kconfig

```diff
@@ -0,0 +1,59 @@
+#
+#For a description of the syntax of this configuration file,
+#see kconfig - language at https: //www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config AIFW
+	bool "AI Framework"
+	default n
+	select HAVE_CXX
+	select HAVE_CXXINITIALIZE if BUILD_FLAT
+	---help---
+		Enables AI Framework.
+
+if AIFW
+
+choice
+    prompt "Select runtime for the AI Framework"
```

Comment:
This is not .c, .h or source file so that you don't need to follow TizenRT coding rules.
But even it is, let's use the same way for indentation in a file.